modules/rostests/apitests/gdi32/GetDIBits.c

```diff
@@ -19,8 +19,22 @@ Test_GetDIBits_xBpp(
     ULONG cjSizeImage, cColors;
     HDC hdc;
 
-    hdc = GetDC(0);
+    hdc = GetDC(NULL);
+    ok(hdc != NULL, "GetDC failed\n");
```

Comment:
Handles are not pointers, I'm not sure it makes sense to use `NULL`.

Reply:
Yes actually it does make sense. Handles are similar to pointers (have the same bit size, etc.)

Reply:
> Handles are not pointers

No: they are defined as pointers.